feat(shell): render correct locked balances on main page

olegshilov opened the pull request.

olegshilov committed:
feat(shell): render correct locked balances, add uc dao fund balance and unbonding amount

GarageInc commented on libs/ui-kit/src/lib/staked-vested-balance.tsx:
```diff
@@ -2,51 +2,83 @@ import { useMemo } from 'react';
+  const { vestedPercent, stakedPercent, daoLockedPercent, unbondingPercent } =
+    useMemo(() => {
+      const all = vested + staked + daoLocked + unbonding;
+      const vestedPercent = (vested / all) * 100;
```
check if 'all' can be ZERO

olegshilov committed:
refactor(shell-ui-kit): improve percentage calculation accuracy

- Add check for division by zero
- Use toFixed to limit decimal places in percentage calculations
- Implement check for total percentage sum equaling 100%
- Add console warning if total percentage is not exactly 100%
